src/features/hotspots/root/WelcomeOverview.tsx

```diff
@@ -38,19 +41,33 @@ const TimeOfDayTitle = ({ date }: { date: Date }) => {
 
 type Props = { accountRewards: CacheRecord<Sum> }
 const WelcomeOverview = ({ accountRewards }: Props) => {
+  const dispatch = useAppDispatch()
   const { t } = useTranslation()
   const { hntBalanceToDisplayVal, toggleConvertHntToCurrency } = useCurrency()
   const [bodyText, setBodyText] = useState('')
-
   const hotspots = useSelector(
     (state: RootState) => state.hotspots.hotspots,
     isEqual,
   )
 
-  const loading = useMemo(() => accountRewards.loading, [
+  const validators = useSelector(
+    (state: RootState) => state.validators.validators.data,
+  )
+  const validatorsLoading = useSelector(
+    (state: RootState) => state.validators.validators.loading,
+  )
+
+  const loading = useMemo(() => accountRewards.loading || validatorsLoading, [
     accountRewards.loading,
+    validatorsLoading,
   ])
 
+  useVisible({
+    onAppear: () => {
+      dispatch(fetchValidators())
```

Comment:
Should we move this to hotspots view where the `fetchAccountRewards()` happens? Or maybe move that here?

Reply:
yes, good call